services/catalog/src/simcore_service_catalog/api/rest/_services.py

```diff
@@ -115,6 +115,7 @@ async def list_services(
     }
     # Non-detailed views from the services_repo database
     if not details:
+        # TODO: what is this used for?
```

Comment:
this is there to make it way faster as it only returns the key/versions available.